README.md

```diff
@@ -1,187 +1,38 @@
 # Warpgrapher
+[![Build Status](https://github.com/warpforge/warpgrapher/workflows/Test/badge.svg)](https://github.com/warpforge/warpgrapher/actions?query=workflow%3A%22Test%22+branch%3Amaster)
+[![GitHub Release](https://img.shields.io/github/release/warpforge/warpgrapher.svg?style=flat)](https://github.com/warpforge/warpgrapher/releases/tag/v0.2.0)
```

Comment:
Does this tag need to be updated to 0.3.0?

Reply:
Fixed.